lang/update_pot.sh

```diff
@@ -32,6 +32,7 @@ xgettext --default-domain="cataclysm-dda" \
          --keyword="gettext_noop" \
```

Comment:
> I just added the one I needed. I don't care about the rest.

Well, you *did* cared for the existing macro `gettext_noop` and removed it from the source, why did you not remove it here as well? This is at least inconsistent.